README.md

```diff
@@ -0,0 +1,217 @@
+
+**`integration`** (required): This must match the name of your integration. (e.g. "my_integration").
+
+**`short_name`**: A more human-readable and abbreviated version of the metric name. For example, `postgresql.index_blocks_read` might be set to `idx blks read`. Aim for human-readability and easy understandability over brevity. Don't repeat the integration name. If you can't make the `short_name` shorter and easier to understand than the `metric_name`, leave this field empty.
+
+#### `requirements.txt`
+
+If you require any additional Python libraries, you can list them here and they will be automatically installed via pip when others use your integration.
+
+#### `test_my_integration.py`
+
+Integration tests ensure that the Datadog agent is correctly receiving and recording metrics from the software you are integrating.
+
+Though we don't require test for each of the metrics collected by your integration, we strongly encourage you to provide as much coverage as possible. You can run the `self.coverage_report()` method in your test to see which metrics are covered.
+
+Here's an example `test_my_integration.py`:
+
+```
+from nose.plugins.attrib import attr
+from checks import AgentCheck
+from tests.checks.common import AgentCheckTest
+
+@attr(requires='my_integration')
+Class TestMyIntegration(AgentCheckTest):
+
+  def testMyIntegration(self):
+    self.assertServiceCheck('my_integration.can_connect',
+                            count=1,
+                            status=AgentCheck.OK,
+                            tags=[host:localhost', 'port:80'])
+    self.coverage_report()
+```
+
+For more information about tests and available test methods, please reference the [AgentCheckTest class in the Datadog Agent repository](https://github.com/DataDog/dd-agent/blob/master/tests/checks/common.py)
+
+### Testing your integration
+
+As you build your check and test code, you can use the following to run your tests:
+
+- `rake lint`: Lint your code for potential errors
+- `rake ci:run[my_integration]`: Run the tests that you have created in your `test_my_integration.py` file.
+- `rake ci:run[default]`: Run the tests you have written in addition to some additional generic tests we have written.
+
+Travis CI will automatically run tests when you create a pull request. Please ensure that you have thorough test coverage and that you are passing all tests prior to submitting pull requests.
+
+
+### Teardown and cleanup
+
+When you have finished building your integration, you can run `rake clean_env` to remove the Python virtual environment.
+
+# Submitting Your integration
+
+Once you have completed the development of your integration, submit a [pull request](https://github.com/DataDog/integrations-extras/compare) to have Datadog review your integration. Once we've reviewed your integration, we will approve and merge your pull request or provide feedback and next steps required for approval.
```

Comment:
i think you should spell this out a bit more. this could be the first open source project they have contributed to. commit to their repo, then submit a pr to integrations-extra. we don't need to list every step but we should point to some instructions